### ISSUE TITLE
Fedora 27 Py3 Build Timing Out after syndic test

### ISSUE THREAD

**Issue by rallytime** [saltstack/salt-jenkins#1025]
The Fedora 27 build for the Python 3 tests is timing out after the following tests:

```
17:42:24        [CPU:0.0%|MEM:12.1%]  ... OK (10.623s)
17:42:24          test_issue_7754 (integration.shell.test_syndic.SyndicTest)
18:12:24 Build timed out (after 30 minutes). Marking the build as aborted.
```

This is happening on both the 2018.3 and develop branches:
- https://jenkinsci.saltstack.com/job/2018.3/view/Python3/job/salt-fedora-27-py3/
- https://jenkinsci.saltstack.com/job/develop/view/Python3/job/salt-fedora-27-py3/

This is not happening on the Python 2 jobs.

@gtmanfred Can you look into this?

**Comment by rallytime** [comment]
This is also happening on Fedora 27 for both branches, Python 3.

**Comment by gtmanfred** [comment]
I think this one is solved by https://github.com/saltstack/salt/pull/49038

Monitoring.

**Comment by rallytime** [comment]
@gtmanfred Hrm, unfortunately, that does not seem to be the case:

https://jenkinsci.saltstack.com/job/2018.3/view/Python3/job/salt-fedora-27-py3/23/console

**Comment by Ch3LL** [comment]
this is fixed in 2018.3.3
